docs/faq.md

```diff
@@ -276,4 +276,26 @@ Note: Windows with Radeon GPUs currently default to 1 model maximum due to limit
 
 ## How does Ollama load models on multiple GPUs?
 
-Installing multiple GPUs of the same brand can be a great way to increase your available VRAM to load larger models.  When you load a new model, Ollama evaluates the required VRAM for the model against what is currently available.  If the model will entirely fit on any single GPU, Ollama will load the model on that GPU.  This typically provides the best performance as it reduces the amount of data transfering across the PCI bus during inference.  If the model does not fit entirely on one GPU, then it will be spread across all the available GPUs.
\ No newline at end of file
+Installing multiple GPUs of the same brand can be a great way to increase your available VRAM to load larger models.  When you load a new model, Ollama evaluates the required VRAM for the model against what is currently available.  If the model will entirely fit on any single GPU, Ollama will load the model on that GPU.  This typically provides the best performance as it reduces the amount of data transferring across the PCI bus during inference.  If the model does not fit entirely on one GPU, then it will be spread across all the available GPUs.
+
+## How can I set the quantization type for the K/V cache?
+
+The K/V context cache can be quantized to significantly reduce memory usage when Flash Attention is enabled.
+
+You can set the quantization type in a number of ways:
+
+1. In the environment using `OLLAMA_CACHE_TYPE_K` and `OLLAMA_CACHE_TYPE_V` which will be the default vault for all models loaded.
+2. In a model's Modelfile using the `cache_type_k` and `cache_type_v` parameters which will be loaded with the model.
+3. In the CLI with `/set parameter cache_type_k <value>` and `/set parameter cache_type_v <value>` which will be used for that session.
+
+While there are [a number of quantization types available](https://github.com/ggerganov/llama.cpp/pull/7527), the most common you might want to choose are:
+
+- `f16` - high precision and memory usage (default).
```

Comment:
Let's limit to `f16`, `q8_0` and `q4_0` - my worry is that setting these would sometimes cause crashes (might be fixed now). Also limiting to 4-bit, 8-bit and 16-bit makes it easy to adopt future backends later

Reply:
No worries, will make that change shortly

Reply:
Done, I also added that f32 is technically an option (although I'm not sure why you'd want to run it?)

Reply:
Thanks! Yes f32 should just be the default as it is today